O365SearchAndIntelligenceConfigurations : Unable to export

Using the following export command on PowerShell 7 with module version 1.24.619.1:
```
Export-M365DSCConfiguration -MaxProcesses $cores -Components O365SearchAndIntelligenceConfigurations -Path $outputPath -FileName $outputFile -CertificateThumbprint $certificateThumbprint -TenantId $TenantId -ApplicationId $ApplicationID
```

I get the following:
Exporting Microsoft 365 configuration for Components: O365SearchAndIntelligenceConfigurations
 
Authentication methods specified:
- Service Principal with Certificate Thumbprint
 
Connecting to {ExchangeOnline}...✅
[1/1] Extracting [O365SearchAndIntelligenceConfigurations] using {}...Partial Export file was saved at: [...]
##[error]You must specify either the Credential or ApplicationId, TenantId and CertificateThumbprint parameters.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,13 @@
 # Change log for Microsoft365DSC
 
+# UNRELEASED
+
+* O365SearchAndIntelligenceConfigurations
+  * Added note that only Credentials are supported for the resource.
+* TeamsOrgWideAppSettings
+  * Added note that only Credentials are supported for the resource.
+    FIXES [#3394](https://github.com/microsoft/Microsoft365DSC/issues/3394)
+
 # 1.25.226.1
 
 * AADConditionalAccessPolicy

--- a/Modules/Microsoft365DSC/DSCResources/MSFT_O365SearchAndIntelligenceConfigurations/readme.md
+++ b/Modules/Microsoft365DSC/DSCResources/MSFT_O365SearchAndIntelligenceConfigurations/readme.md
@@ -3,3 +3,5 @@
 ## Description
 
 This resource configures the Search And Intelligence configuration settings.
+
+**This resource currently only supports Credential authentication. Please refer to [Microsoft Graph REST API Beta Reference](https://learn.microsoft.com/en-us/graph/api/organizationsettings-list-contactinsights?view=graph-rest-beta&tabs=http) for further information about the supported authentication methods.**

--- a/Modules/Microsoft365DSC/DSCResources/MSFT_TeamsOrgWideAppSettings/readme.md
+++ b/Modules/Microsoft365DSC/DSCResources/MSFT_TeamsOrgWideAppSettings/readme.md
@@ -4,3 +4,5 @@
 ## Description
 
 This resource configures Org-Wide App Settings for Teams.
+
+**This resource currently only supports Credential authentication. Please refer to [Teams PowerShell Application Authentication](https://learn.microsoft.com/en-us/microsoftteams/teams-powershell-application-authentication) for further information.**

